src/shogun/labels/MulticlassLabels.cpp

```diff
@@ -134,3 +134,16 @@ int32_t CMulticlassLabels::get_num_classes()
 	SGVector<float64_t> unique=get_unique_labels();
 	return unique.vlen;
 }
+
+CLabels* CMulticlassLabels::shallow_subset_copy()
```

Comment:
BTW for both of those methods: You either need to hide them from SWIG or put a `%newobject` directive in the interface file